src/ui/public/vislib/visualizations/gauges/simple.js

```diff
@@ -100,6 +100,37 @@ export function SimpleGaugeProvider() {
       return this.colorFunc(labels[bucket]);
     }
 
+    wrapText(text, width) {
+      text.each(function () {
+        const text = d3.select(this);
```

Comment:
Maybe `const text` should be renamed since there is a naming collision with the `wrapText` method parameter `text`.